docs/docs_requirements.txt

```diff
@@ -23,12 +23,17 @@ PyYAML==3.11
 South==1.0.2
 python-ldap==2.4.19
 
-# Deis client requirements
+# Deis client requirements, copied from client/requirements.txt
 docopt==0.6.2
+ndg-httpsclient==0.3.3
+pyasn1==0.1.7
+pyOpenSSL==0.15.1
 python-dateutil==2.4.2
+#PyYAML==3.11
```

Comment:
Because we have PyYAML elsewhere in the same file, and `pip` will refuse and quit when it sees a duplicate requirement. So this is commented out to document that the client requires it (and so this copied section matches `client/requirements.txt` better.

(And `docs_requirements.txt` has to be maintained separately because Deis docs are published at ReadTheDocs.org, and its admin interface allows us to specify exactly one requirements file. So we still have some copy-and-pasting maintenance we have to do.)